control/tests/timeresp_test.py

```diff
@@ -303,6 +339,73 @@ def test_step_info(self):
                                    [Strue[k] for k in Sktrue],
                                    rtol=rtol)
 
+ # tolerance for all parameters could be wrong for some systems ej: discrete systems time parameters tolerance
+    # could be +/-dt
```

Comment:
```suggestion
    # tolerance for all parameters could be wrong for some systems
    # discrete systems time parameters tolerance could be +/-dt
```

Reply:
Im just copy and paste an old test, but I am not satisfied with the solution, because use the same tolerance value for all the output, and this could be not right. If you have any suggestion are well come

Reply:
The comment/suggestion is just about the broken formatting